salt/lax/bot-lax-adaptor.sls

```diff
@@ -149,22 +150,30 @@ bot-lax-uwsgi-conf:
             - bot-lax-adaptor-install
             - bot-lax-writable-dirs
 
-{% set apiprotocol = 'https' if salt['elife.cfg']('cfn.outputs.DomainName') else 'http' %}
-{% set apihost = salt['elife.cfg']('project.full_hostname', 'localhost') %}
-
-uwsgi-bot-lax-adaptor:
+bot-lax-uwsgi-upstart:
     file.managed:
         - name: /etc/init/uwsgi-bot-lax-adaptor.conf
         - source: salt://lax/config/etc-init-uwsgi-bot-lax-adaptor.conf
         - template: jinja
         - mode: 755
```

Comment:
not sure what the upstart requirements were, but systemd has a verification command (because it doesn't check syntax at runtime and fails silently!) and it complains if the service unit is executable